app/jobs/send_purge_warnings_job.rb

```diff
@@ -14,8 +14,8 @@ def perform(*_args)
 
     # Loop through and send each admin information about their purge eligible monitorees
     recipients.each do |user|
-      # Skip for USA admins
-      next if user.jurisdiction&.name == 'USA'
+      # Skip for USA admins and locked users
+      next if user.jurisdiction&.name == 'USA' || !user.locked_at.nil?
```

Comment:
I actually think it might be better if this check is done here:
```
recipients = User.where(role: [Roles::ADMIN, Roles::SUPER_USER])
```

like so:
```
recipients = User.where(role: [Roles::ADMIN, Roles::SUPER_USER], locked_at: nil)
```
The USA check would be better there as well to be honest.